Src/Bonzai.Blazor.Bootstrap.Documentation/Client/Pages/1_Practice.razor

```diff
@@ -0,0 +1,129 @@
+﻿@page "/hello-world"
+
+
+<body class="@page_class vw-100 vh-100">
+
+    <h1 class="@helloWorld_class display-1"> Hello World </h1>
+
+    <div>
+        
+        <!-- Dropdown -->
+        <select name="TextColour_dropdown" 
+                @onchange="StringColourChange">
```

Comment:
This works but it can be a lot simpler. Take a look at data binding in blazor. https://docs.microsoft.com/en-us/aspnet/core/blazor/components/data-binding?view=aspnetcore-6.0

If you set the option's value property to the class name this just becomes `@bind=helloWorld_class` and you can get rid of the `StringColourChange` function.